frontend/src/pages/Dashboard/index.css

```diff
@@ -1,9 +1,26 @@
 @import '~antd/dist/antd.css';
 
 .column {
-  font-size: 25px;
```

Comment:
![Screenshot 2021-03-12 at 10 43 38 AM](https://user-images.githubusercontent.com/15646062/110884285-da9ed800-831f-11eb-9656-117976646f76.png)

The web framework is responsive, any idea to adapt the ant table?

Reply:
We could use a relative font size

Reply:
FYI: https://stackoverflow.com/questions/65508782/how-to-make-ant-design-table-responsive